README.md

```diff
@@ -22,16 +22,16 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [Portfolio: desktop version] <a name="about-project"></a>
+# 📖 [Portfolio: Deployment of desktop version] <a name="about-project"></a>
 
-mv_portfolio: desktop version!
+mv_portfolio1: Deployment of desktop version!
 
-**Portfolio: desktop version** A project for finishing the desktop portfolio app!
+**Portfolio: Deployment of desktop version** A project for deployment of mobile & desktop portfolio app!
 
 ## 🛠️ Built With <a name="built-with">
 ### Tech Stack <a name="tech-stack"></a>
 - HTML
-- CSS
+- CSS & Media Queries
 
 </a>
 
```

Comment:
- [ ] Good job by providing a working deployed link of your project. However as required for the project, kindly ensure that your README is updated to include a link to the online version. That means you should add the `Live section` you removed earlier that include the deployed link.
```
<!-- LIVE DEMO -->

## 🚀 Live Demo <a name="live-demo"></a>

> Add a link to your deployed project.

- [Live Demo Link](https://jcmunav63.github.io/mv_portfolio1/)
```
For a short remind where you should put the `Live Demo` section, just take a look again on the **[template](https://github.com/microverseinc/readme-template)**

Reply:
Dear reviewer @tresorsawasawa:

I included the live demo link that you request in the README.md file, under the section **"Deployment (Github)"**. I don't know if this is enough to get this project approved. Thank you.